tests/UI/pages/FO/checkout/index.js

```diff
@@ -16,6 +16,8 @@ class Checkout extends FOBasePage {
     this.termsOfServiceModalDiv = '#modal div.js-modal-content';
     this.paymentConfirmationButton = `${this.paymentStepSection} #payment-confirmation button:not([disabled])`;
     this.shippingValueSpan = '#cart-subtotal-shipping span.value';
+    this.noPaymentNeededElement = `${this.paymentStepSection} div.content > p`;
```

Comment:
```suggestion
    this.noPaymentNeededElement = `${this.paymentStepSection} div.content > p.cart-payment-step-not-needed-info`;
```

I found out where you can add class for that selector to look better, it's in this file `themes/classic/templates/checkout/_partials/steps/payment.tpl`, line 17.

Reply:
Ping @tswfi : a feedback from @boubkerbribri

Reply:
yup, I'm checking the comments (and trying to get the test run again :)
Already merged the obvious typo fixes etc and rebased this branch.

Reply:
I did think about adding the class already way back when I made the pull request initially, but decided to try to make minimal changes.
And yes, this is the correct way to get the right element. Added the class and fixed the selector in e0756fb